rotkehlchen/tests/unit/decoders/kyber.py

```diff
@@ -0,0 +1,223 @@
+import pytest
+from hexbytes import HexBytes
+
+from rotkehlchen.accounting.structures import (
+    Balance,
+    HistoryBaseEntry,
+    HistoryEventSubType,
+    HistoryEventType,
+)
+from rotkehlchen.chain.ethereum.decoding.decoder import EVMTransactionDecoder
+from rotkehlchen.constants.assets import A_CRV, A_ETH, A_USDC
+from rotkehlchen.db.ethtx import DBEthTx
+from rotkehlchen.db.filtering import HistoryEventFilterQuery
+from rotkehlchen.fval import FVal
+from rotkehlchen.types import EthereumTransaction, Location, deserialize_evm_tx_hash
+from rotkehlchen.user_messages import MessagesAggregator
+
+
+@pytest.mark.parametrize('ethereum_accounts', [['0x6d379cb5BA04c09293b21Bf314E7aba3FfEAaF5b']])  # noqa: E501
+def test_kyber_legacy_old_contract(database, ethereum_manager):
+    # Data for trade
+    # https://etherscan.io/tx/0xe9cc9f27ef2a09fe23abc886a0a0f7ae19d9e2eb73663e1e41e07a3e0c011b87
```

Comment:
Use a docstring here
```suggestion
    """Data for trade taken from:
    https://etherscan.io/tx/0xe9cc9f27ef2a09fe23abc886a0a0f7ae19d9e2eb73663e1e41e07a3e0c011b87
    """
```